_data/meltano/utilities/superset/apache.yml

```
@@ -9,37 +9,123 @@ variant: apache
 pip_url: apache-superset==1.5.0 markupsafe==2.0.1 # `markupsafe` is listed explicitly to resolve https://github.com/apache/superset/issues/19150
 settings:
 - name: ui.bind_host
-  label: Host
+  label: UI Bind Host
   value: "0.0.0.0"
+  description: Host used by `meltano invoke superset:ui`. Used in the `gunicorn` `--bind` option along with `ui.port`.
 - name: ui.port
-  label: Port
+  label: UI Port
   value: 8088
+  description: Port used by `meltano invoke superset:ui`. Used in the `gunicorn` `--bind` option along with `ui.bind_host`.
 - name: ui.timeout
-  label: Timeout 
+  label: UI Timeout
   value: 60
-  description: UI Timeout in seconds.
+  description: Timeout in sections, used by `meltano invoke superset:ui`. Corresponds to the `gunicorn` `--timeout` option
 - name: ui.workers
-  label: Workers
+  label: UI Workers
   value: 4
+  description: Workers used by `meltano invoke superset:ui`. Corresponds to the `gunicorn` `--workers` option
 - name: SQLALCHEMY_DATABASE_URI
   label: SQLAlchemy Database URI
   value: sqlite:///$MELTANO_PROJECT_ROOT/.meltano/utilities/superset/superset.db
+  description: |
+    Superset metadata database connection string. Equivalent to the `SQLALCHEMY_DATABASE_URI` setting in `superset_config.py`.
+
+    SQLite is used by default, but in production PostgreSQL or MySQL is more appropriate. See the [Superset documentation](https://superset.apache.org/docs/installation/configuring-superset/#using-a-production-metastore) for more information. To add a database driver like `psycopg2` or `mysqlclient`, refer to the [Add additional database drivers](#add-additional-database-drivers) section above.
 - name: SECRET_KEY
   label: Secret Key
   kind: password
   value: thisisnotapropersecretkey
+  description: App secret key used for securely signing the session cookie and encrypting sensitive information on the database. Equivalent to the `SECRET_KEY` setting in `superset_config.py`.
 commands:
   ui:
     executable: gunicorn
     args: --bind $SUPERSET_UI_BIND_HOST:$SUPERSET_UI_PORT --timeout $SUPERSET_UI_TIMEOUT --workers $SUPERSET_UI_WORKERS superset.app:create_app()
-    description: Start the Superset UI. Will be available on the configured `ui.bind_host` and `ui.port`, which default to `http://localhost:4000`
+    description: Start the Superset UI. Will be available on the configured `ui.bind_host` and `ui.port`, which default to `http://localhost:8088`
   create-admin:
     args: fab create-admin
     description: Create an admin user.
-  load_examples:
+  load-examples:
     args: load_examples
     description: Load examples.
 logo_url: /assets/logos/utilities/superset.png
 definition: is a modern, enterprise-ready business intelligence web application. **It requires Meltano version 2.0.**
+maintenance_status: active
```

```suggestion
maintenance_status: active
keywords:
- analysis
```

Needed for the index page listing